### ISSUE TITLE
Problem with denoising

### ISSUE THREAD

**Issue by EliaBullano** [PENGLU-WashU/IMC_Denoise#36]
Hi, I've 14 file.tiff with 22 channels, identical in the name and in the position, but with different noise, i use the script with multi_channel_tiff over all the channels but always give me loss nan, i've tried to change train_learning_rate but the result are or nan or a loss very high so the denois isn't good. Could you give a preset code for denoising multi_channel_tiff and an advise for the nan loss?

Thank you.
